.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
can you share why this was added?

Reply:
Bizarrely I have no clue. Initially assumed it was from trying different things to fix the npm install error that was coming up when I was first trying to build the dev container, but seems like I merged that to master without this file being there. Definitely didn't manually add it because I haven't consciously worked with/really thought about github's "dependabot" before. Will delete and pay more attention to staged changes in the future !